msg/output_control.msg

```diff
@@ -0,0 +1,95 @@
+uint64 timestamp		 # time since system start (microseconds)
+uint64 timestamp_sample	 # the timestamp the data this control response is based on was sampled
+
+uint8 NUM_OUTPUT_CONTROL_GROUPS = 4 # Max number of instances of this topic
+uint8 MAX_ACTUATORS = 16  # Max actuators available in a single topic publication
+
+uint8 n_outputs
+uint16[16] function       # Actuator function being controlled
```

Comment:
Why do we need it in this topic ? 
I would think topic ID + value index is sufficient to know which function is being set.
Ex: output_control_ca.value[0] is function FUNCTION_CA0.